frontend/server/src/Controllers/User.php

```diff
@@ -1429,6 +1429,53 @@ public static function apiProfile(\OmegaUp\Request $r): array {
         );
     }
 
+    /**
+     * When logged user is not allowed to see the information of another user
+     * we just need to send an array with empty profile
+     *
+     * @return UserProfileInfo
+     */
+    private static function getEmptyUserProfile(
```

Comment:
nit:

```suggestion
    private static function getPrivateUserProfile(
```

esto está _casi_ vacío, pero sí tiene unas cuantas cosas.

Reply:
Listo.